Задание_8.4/Задание_8.4/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+
+namespace Задание_8._4
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите последовательность чисел");
+            double n;
+            n = Convert.ToDouble(Console.ReadLine());
+            Console.WriteLine("Введите делитель");
+            double k;
+            k = Convert.ToDouble(Console.ReadLine());
+            double temp, a;
```

Comment:
И здесь достаточно использовать целые числа.
```suggestion
            int temp, a;
```